### ISSUE TITLE
SKSE64Plugins alpha 6 CTD issue

### ISSUE THREAD

**Issue by Saqer80** [expired6978/SKSE64Plugins#8]
I've noticed that I cannot load my saved game and it will CTD in load menu after installing racemenu plugins alpha 6 release and making my character with it... But when I delete the plugins (skee.dll hudextension.dll) the game will load normally.

**Comment by PsychoWookiee** [comment]
Same issue here. Every time I try to load a save with this installed, it'll go to the load screen and then CTD, but it works fine without this mod installed. This is only the most recent update to SKSE and Racemenu Alpha 6. Not sure if it helps, but I've attached my load order below.


[plugins.txt](https://github.com/expired6978/SKSE64Plugins/files/1929700/plugins.txt)

**Comment by Saqer80** [comment]
I can't afford to lose Racemenu now especially after I made dual sheath redux works perfectly with it... I'm still investigating this issue and I think I'm kinda close to figuring it out... I'm thinking its something to do with other hud mods that conflicting with Racemenu plugins like a matter of time, iHud and SkyHud etc... I'm gonna uninstall them all and try a new game then saving and loading again after few mins of gameplay to see if it's gonna CTD or not.

**Comment by PsychoWookiee** [comment]
I'm posting my MO2 modlist here. Maybe you can look through it and see what mods we have in common in order to identify the conflict. Right off the bat, I can tell you that I am using iHud but not SkyHud or AMoT. I'm not using DSR either, FYI.

[modlist.txt](https://github.com/expired6978/SKSE64Plugins/files/1929917/modlist.txt)

**Comment by aers** [comment]
Its the new version of XPMSSE (4.3/4.31) causing skee to crash, probably because of some untested/unimplemented skee function.

**Comment by PsychoWookiee** [comment]
Just tested and can confirm, removing the latest XPMSSE but keeping Racemenu works, or vice versa, but having both on crashes when loading a save. So aers is correct.

**Comment by Saqer80** [comment]
You are right!... I've tried that also and it was fine... What a waste!... I hope it will get updated soon.

**Comment by Shad0wlife** [comment]
Might actually not be an SKEE sided issue. The patch notes of 4.3 say something about 
"- esp without RaceMenu components added"
so maybe you have the wrong esp installed? Or something else from the XMPSSE esp breaks the mod?

**Comment by aers** [comment]
Nah there's definitely a crash on save game load. I tracked down the crash in skee and reported it to expired in discord; he'll fix it when he has the free time.

**Comment by fruttyx** [comment]
Thanks aers.

**Comment by aers** [comment]
If you're capable of building skee yourself setting the preprocessor definition _NO_REATTACH will flip the code in BodyMorphInterface that is causing the crash on load. You'd also have to update the SKSE64 source because the offsets in NiRTTI.cpp & NiExtraData.cpp are out of date.

tl;dr You're probably better off waiting until Expired can take a look

**Comment by Saqer80** [comment]
Is there another mod that uses Skee64.dll other than XPMSSE ver4.31?... I want to see if the save game gonna crash again or not.

**Comment by DragonJoe** [comment]
Will simply reverting to v4.20 of xpmsse fix the issue, at least until it's fixed?

**Comment by darkconsole** [comment]
@DragonJoe did for me.

**Comment by CycloneGamer** [comment]
How I fixed the CTDs on loading a save with XPMSSE v4.31 and RaceMenu Alpha 6.

While installing the latest version of XPMSSE, during the FOMOD installation, select the following options:
1. a. Character Creation: MCM.
    b. Weapon Style Randomizer for NPCs: None.
2. Done.

**Comment by thmsndk** [comment]
I appear to be getting CTD with alpha 6 aswell,going back to XPMSSE v4.20 does not appear to work for me. and trying what CycloneGamer mentioned did not appear to do anything either. I guess i'm doing something wrong?

**Comment by Sasha1378** [comment]
I have some weird crashes after CycloneGamer's fix still... After I boot up my computer next morning after saving, my saves won't load up to game, even though they did the previous day. I can always load the save without SKSE, but not with it. I can still play a new game with SKSE though. Very strange.

**Comment by dizzy223** [comment]
I'm not sure what is causing it but the game crashes on every character for Racemenu when loading, quick, auto and even normal saves cause a CTD.

**Comment by padely** [comment]
In my case, the peoblem is skee alpha 6. I tried to disable all of my mods and the game CTD when loading a save game. Then I tried to delete skee plugins and the game run fine.
I tried to use skee alpha 5 and it also CTD. Alpha 4 is working fine for me. My game version is 1.5.39 and SKSE warns me about compatibility issue, but it works.

**Comment by wright3258** [comment]
I've been able to use the latest skee alpha 6 with the previous version of XPMS 4.2 with great success. All latest features work. You just don't get the styles included with the latest XPMSE. You also don't have the options related to choosing ECE, Racemenu or Dealy Mutilation. However, all of these mods still work with the XPMSE 4.2 skeleton.

**Comment by Papajack1955** [comment]
I am trying to find the latest issue of Racemenu to install into my SE game. I have the latest SKYUI in, but this Alpha 6 leads me to Oldrim Nexus. I though Oldrim RaceMenu wouldn't work in SE?

**Comment by AngryPillow** [comment]
Those of you attempting to make this functional in SE should also resave both racemenu esp's in the SE creation kit, as the oldrim page plugins are using old form 43.  While not necessarily a contributing factor to this particular issue as it seems to be SKEE-related, it's still a good idea as using old-form plugins can lead to save corruption.

**Comment by TheDabKing** [comment]
SOLVED!

Just updated XPMSSE to 4.32 so now i can use both RaceMenu and XPMSSE !!!!!

**Comment by capella07** [comment]
@TheDabKing, What did you do to get them to work?

**Comment by TheDabKing** [comment]
@capella07 I updated XPMSSE to 4.32

**Comment by capella07** [comment]
Okay, so I'm reinstalling everything (not related to SKSE/SKEE/XPMSSE, etc)
so I'll give it a try. You didn't change anything other than upgrade from
4.2 to 4.32? I'm still wary of going to 4.32, as there are recent posts on
the XPMSSE of people still having infinite load/crash issues with 4.32...
______________________
Mark Yorkovich

**Comment by TheDabKing** [comment]
@capella07 I was on 4.31 and had CTD (Crash To Desktop). I only upgraded to 4.32 with NMM (Nexus Mod Manager) and that's all.

**Comment by capella07** [comment]
@TheDabKing, Okay, thanks. I'm still not sure if I'll upgrade, but I'm on a
fresh game, so I suppose I wouldn't be losing much if I run into save game
corruption issues or something.
______________________
Mark Yorkovich

**Comment by wooby6** [comment]
Or you can all just install [SSE Engine Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/17230) 
until expired get around to fixing it 
(it fixes all Load CTD issues, And I'm running Alpha6 + Racemenu + XPMSSE 4.32)

**Comment by capella07** [comment]
@wooby6, Yeah - I already have that installed. I just don't remember at
this point if I first installed that before or after I was getting CTDs
with XPMSSE v4.31.

Also, I'm running ECE, not Racemenu. It was my understanding that the
crashes were due to ECE and SKEE, but am I wrong on that?
______________________
Mark Yorkovich

**Comment by wooby6** [comment]
The crashes are due in part to the buggy alpha of SSE, So it's been recommended to install [SSE Engine Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/17230) across many forums, this does indeed solve the issue with racemenu (in fact it even mentions this on the nexus page), but I'm not sure about ECE

**Comment by PsychoWookiee** [comment]
I use SSE Engine Fixes with Racemenu Alpha 6 and that fixed the loading CTDs but I would still crash after a few minutes of playing whenever I had XPMSSE v4.31 or v4.32, that went away when I reverted back to 4.20. So, in my case, I'm very content with 4.20 right now until Racemenu itself gets properly updated.

**Comment by wooby6** [comment]
@PsychoWookiee that's  a different bug then what the OP put, this issue is only for "load game CTD"

**Comment by HenryOldman** [comment]
It's pretty sure that this plugin will definitely causes CTD when you're trying to load a save game. And the price to solve this problem is unsure as just unload it or using the fix patch existed on Nexus ---- which may cause other problems (watch the BUGs report area you will see ). 
My current XPMSEE version is 4.32 and I use ECE.
Waiting for update.

**Comment by Neondaze** [comment]
As has previously said, SSE Engines fixes resolves this issue.  I am using alaph 6 with racemenu and the latest xm32 msse skeleton. I have no crashes on loading save games.
